File: c/exerciciosChat/02.c
#include <stdio.h>

int maior(int x , int y , int z){
      if(x>y && x>z){
    printf("%i é maior que %i e % i",x,y,z);
  }else if(y>x && y>z){
    printf("%i é maior que %i e %i",y,x,z);
  }else if(z>x && z>y){
    printf("%i é maior que %i e %i",z,y,x);
  }else{
    printf("os numeros são iguais");
  }

}


int main(){
  int x =5;
  int y = 6;
  int z = 7;
  maior(x,y,z);
}

File: javascript/exercicios/02.js
//2)Os triângulos podem ser classificados em 3 tipos quanto ao tamanho de seus lados:
//Equilátero: Os três lados são iguais. Isósceles: Dois lados iguais. Escaleno: Todos os lados são diferentes.
//Crie uma função que recebe os comprimentos dos três lados de um triângulo e retorne sua classificação quanto
//ao tamanho de seus lados. (Neste exemplo deve-se abstrair as condições matemáticas de existência de um
//triângulo).

function triangulo(a, b, c) {
    if (a == b && a == c) {
        return "Equilatero"
    } else if (a != b && a == c || a == b && a != c) {
        return "Isosceles"
    } else if (a != b != c) {
        return "Escaleno"
    } else {
        return "algo deu errado"
    }
}
console.log(triangulo(2,2));